### ISSUE TITLE
SNOW-885065: sktime package not found in Anaconda channel when it is

### ISSUE THREAD

**Issue by sfc-gh-xbonnamy** [snowflakedb/snowflake-cli#292]
### Python version

Python 3.8.15

### Platform

macOS-10.16-x86_64-i386-64bit

### What happened

Running $snow -v snowpark package lookup sktime
Reports that sktime' not found in Snowflake anaconda channel
When it actually is

To check sktime is available, running the following SQL:
select * from <database>.information_schema.packages
    where true
        and language = 'python'
        and package_name = 'sktime'
;

### Console output

```Shell
INFO 'sktime' not found in Snowflake anaconda channel...
```

### How to reproduce

Run the below command
snow -v snowpark package lookup sktime

**Comment by sfc-gh-jhollan** [comment]
strange - in this case it’s because the CLI is looking at Anaconda Metadta (at https://repo.anaconda.com/pkgs/snowflake/channeldata.json) to know what’s in the channel, and sktime does not, in fact, show up. But I do see it’s actually there. I can ping Anaconda and figure out why there would be a mismatch between channeldata and the actual channel

**Comment by sfc-gh-twhite** [comment]
I have seen this mismatch before as well. I have seen certain cases where things differ slightly based on the client's operating system, which could require parsing through the `subdirs` key. There's a reference to this behavior occurring in an error handle.

https://github.com/Snowflake-Labs/snowcli/blob/b7df2ad63d9480a6287200c467c6c26968d07077/src/snowcli/utils.py#L406C1-L413C27

Can we use pip's platform argument on the install command? https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-platform?

**Comment by sfc-gh-turbaszek** [comment]
Seems to no longer be an issue
```
➜ snow snowpark package lookup sktime                                           
Package `sktime` is available in Anaconda. Latest available version: 0.24.1.
```

Closing for now
